src/core/drive/form_submission.ts

```diff
@@ -56,6 +70,24 @@ export class FormSubmission {
     return expandURL(this.action)
   }
 
+  get body() {
+    if (this.enctype == FormEnctype.applicationXWWWFormUrlEncoded) {
```

Comment:
```suggestion
    if (this.method == FetchMethod.get || this.enctype == FormEnctype.applicationXWWWFormUrlEncoded) {
```

Why don't we also want the `get` check here? Feels a little unsafe/unclear to me to infer this purely based on the `enctype`?